### DISCUSSION TITLE
DB2 Generate DDL also gives all views

### DISCUSSION THREAD

**Discussion by twielens** [dbeaver/dbeaver#14955] [Q&A]
Hello,

We've been using DBeaver for quite a while and now we finally obtained the rights to use the generate DDL function (DB2LOOK) which is working but the problem is that when I try to get the DDL for a table I also get a lot of views, even views from different schema's. Is there a way to add the -noview option to the db2look command that is used by dbeaver?

I can't put my finger on which specific views it generates the DDL, but it's a whole lot.

Kind regards,

Tim

**Comment by LonwoLonwo** [comment]
Hello @twielens 

There are no options for DDL generating for DB2 in DBeaver.
I created a new issue for this: https://github.com/dbeaver/dbeaver/issues/14959

**Reply by twielens** [reply]
It would be great to have added options indeed, allthough I'm quite curious if other people also have this problem? I'm trying to understand why it's picking up all those views and why specifically those views, because they're not always from the same schema for instance.

**Comment by LonwoLonwo** [comment]
Unfortunately, I don't have answers to your questions. DBeaver uses standard queries for DDL generating, like DB2LK_GENERATE_DDL, and shows the result then.
